scalameter-core/src/main/scala/org/scalameter/utils/ClassPath.scala

```diff
@@ -20,6 +20,12 @@ class ClassPath private (val paths: Seq[File]) extends Serializable {
     new ClassPath(location +: paths)
   }
 
+  /** Returns classpath without first location.
```

Comment:
I would just update the comment. I would also add a method that checks if the classpath is empty (i.e. has no paths). Otherwise, using this function always requires `try-catch` blocks.